app/models/comment.rb

```diff
@@ -0,0 +1,6 @@
+class Comment < ApplicationRecord
+  belongs_to :post
+  def update_comments_counter_for_post
+    post.update(comments_counter: post.comments.count)
+  end
+end
```

Comment:
- [ ] Comment belongs to a user. Please make sure to set up an association between comment and author(user) on this model.

Reply:
Hi @thecodechaser you right, I did fix it